test/RelayHubGasCalculations.test.ts

```diff
@@ -23,28 +23,18 @@ const TestRecipient = artifacts.require('TestRecipient')
 const TestPaymasterVariableGasLimits = artifacts.require('TestPaymasterVariableGasLimits')
 const TestPaymasterConfigurableMisbehavior = artifacts.require('TestPaymasterConfigurableMisbehavior')
 
-function correctGasCost (buffer: Buffer, nonzerocost: number, zerocost: number): number {
-  let gasCost = 0
-  for (let i = 0; i < buffer.length; i++) {
-    if (buffer[i] === 0) {
-      gasCost += zerocost
-    } else {
-      gasCost += nonzerocost
-    }
-  }
-  return gasCost
-}
+require('source-map-support').install({ errorFormatterForce: true })
```

Comment:
put it in truffle.js, otherwise tests that run before this will probably still have distorted stack traces